src/test/java/uk/gov/ons/ctp/response/casesvc/IACServiceStub.java

```diff
@@ -18,6 +19,7 @@ public class IACServiceStub {
 
   public void createIACStub() {
     log.info("Stubbing IAC Service POST /iacs endpoint");
+    configureFor("localhost", 18002);
```

Comment:
This is the way to tell the framework to make requests to `18002` we're running the server on, instead of the default `8080`